pipeline/inputs/kafka.md

```diff
@@ -0,0 +1,75 @@
+# Kafka
+
+Kafka input plugin allows to subscribe to one or more topics and collect messages from an [Apache Kafka](https://kafka.apache.org/) service. This plugin use the official [librdkafka C library](https://github.com/edenhill/librdkafka) \(built-in dependency\)
```

Comment:
I like separate lines per sentence just to make it easier to review - you can only comment by line so if you put a whole paragraph on one line it's hard!